ui/main.qml

```diff
@@ -25,7 +25,7 @@ StatusWindow {
     id: applicationWindow
     objectName: "mainWindow"
     minimumWidth: 900
-    minimumHeight: 680
+    minimumHeight: 800
```

Comment:
Not sure if this has been discussed with design/rabbits, but it feels like this is the wrong way to tackle this.
Shouldn't the keycard modal only take up at most the height of the app window - the agreed upon margins (just like all the other modals do)?

Increasing the app's minimum height seems like a hack to me. 

But again, if this has been discussed with design/rabbits you can ignore this comment

Reply:
@PascalPrecht no, it has not been discussed, I wrote about it in the Status app (general channel) and just created a PR, for any case (we can just throw it away if we don't want this change, it's a simple change).

The thing why I am asking for this... look at figma design for keycard flows, actually take a look at this particular view (but it's not the only one with that height):
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=9164%3A192804

you will see that the height of that popup is way bigger than 640px which is the min app height at the moment and because of that popup looks cut if you run such flows while your app is resizes to min app height (look at screenshots above).

Based on that, assuming that designers set the height of that popup, I guess that they wouldn't say that the min app height is 600px?

Reply:
Not against it, but now we have this very non-standard 900x800 resolution. Not sure what this particular modal is using, however StatusDialog does exactly that - uses all available height minus the standard padding around (64px)